gexpect.go

```diff
@@ -378,27 +378,26 @@ func (expect *ExpectSubprocess) Interact() {
 }
 
 func (expect *ExpectSubprocess) ReadUntil(delim byte) ([]byte, error) {
-	join := make([]byte, 1, 512)
+	join := make([]byte, 0, 512)
 	chunk := make([]byte, 255)
 
 	for {
-
 		n, err := expect.buf.Read(chunk)
 
-		if err != nil {
-			return join, err
-		}
-
```

Comment:
Dunno, it is not a change I have made, just some fix from upstream. But probably yes, would need a test for it to know.